job_dsl/rpc_gating_merge_trigger.groovy

```diff
@@ -1,7 +1,7 @@
 // Can't use env.FOO = {FOO} to transfer JJB vars to groovy
 // as this file won't be templated by JJB.
 // Alternative is to use parameters with JJB vars as the defaults.
-library "rpc-gating@${RPC_GATING_BRANCH}"
+library "rpc-gating-master"
 common.globalWraps(){
 
   // We do not want to trigger any of these
```

Comment:
nit: I guess this check is no longer required and could be removed.

Reply:
I think this check can stay, as we still want to avoid triggering from pushes to non-master branches, even if the groovy lib is always loaded from master.